tests/integration/logging/monolog2/test_monolog_labels_forwarding_exclude_05.php

```diff
@@ -0,0 +1,216 @@
+monolog2.NOTICE: notice []
+monolog2.WARNING: warning []
+monolog2.ERROR: error []
+monolog2.CRITICAL: critical []
+monolog2.ALERT: alert []
+monolog2.EMERGENCY: emergency []
+*/
+
+/*EXPECT_METRICS
+[
+  "?? agent run id",
+  "?? timeframe start",
+  "?? timeframe stop",
+  [
+    [{"name": "DurationByCaller/Unknown/Unknown/Unknown/Unknown/all"},            [1, "??", "??", "??", "??", "??"]],
+    [{"name": "DurationByCaller/Unknown/Unknown/Unknown/Unknown/allOther"},       [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Logging/lines"},                                                   [8, "??", "??", "??", "??", "??"]],
+    [{"name": "Logging/lines/ALERT"},                                             [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Logging/lines/CRITICAL"},                                          [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Logging/lines/DEBUG"},                                             [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Logging/lines/EMERGENCY"},                                         [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Logging/lines/ERROR"},                                             [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Logging/lines/INFO"},                                              [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Logging/lines/NOTICE"},                                            [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Logging/lines/WARNING"},                                           [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransaction/all"},                                            [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransaction/php__FILE__"},                                    [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransactionTotalTime"},                                       [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransactionTotalTime/php__FILE__"},                           [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/PHP/Monolog/enabled"},                      [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/PHP/package/monolog/monolog/2/detected"},           [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/library/Monolog/detected"},                         [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/LocalDecorating/PHP/disabled"},             [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/Forwarding/PHP/enabled"},                   [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/Metrics/PHP/enabled"},                      [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/Labels/PHP/enabled"},                      [1, "??", "??", "??", "??", "??"]]
+  ]
+]
+*/
+
+
+/*EXPECT_LOG_EVENTS
+[
+    {
+      "common": {
+        "attributes": { }
+      },
```

Comment:
For other reviewers - sending `"common"` object with empty `"attributes"` is an existing functionality.